templates/log-in.html

```diff
@@ -0,0 +1,23 @@
+<html>
+
+<head>
+    <title>Log In</title>
+    <style>
```

Comment:
Should we transfer this block into a .css file so the login page and signup page will have the same look?